backendApi/src/main/java/com/virtuslab/gitmachete/backend/api/OngoingRepositoryOperation.java

```diff
@@ -1,5 +1,5 @@
 package com.virtuslab.gitmachete.backend.api;
 
 public enum OngoingRepositoryOperation {
-  NO_OPERATION, CHERRY_PICK, MERGING, REBASING, REVERTING
+  NO_OPERATION, CHERRY_PICKING, MERGING, REBASING, REVERTING, APPLY, BISECTING
```

Comment:
`APPLYING` for consistency

Reply:
Ok, but I'm not sure if it's correct: in `RepositoryState` from JGit it is `APPLY` when others are with `-ING` :thinking: